config/index.js

```diff
@@ -436,7 +436,9 @@ var conf = convict({
         'fx_firstrun_v2',
         'fx_desktop_v1',
         'fx_desktop_v2',
-        'fx_desktop_v3'
+        'fx_desktop_v3',
+        'fx_ios_v1',
```

Comment:
According to content-server, there's also an `fx_ios_v2` context that we should probably add to this list.